doc/paper/paper.bib

```diff
@@ -0,0 +1,100 @@
+
+@Misc{pywavelets_zenodo,
+  author       = {Gregory R. Lee and
+                  Ralf Gommers and
+                  Kai Wohlfahrt and
+                  Filip Wasilewski and
+                  Aaron O'Leary and
+                  Holger Nahrstaedt and
+                  David Menéndez Hurtado and
+                  Thomas Arildsen and
+                  Helder Oliveira and
+                  Ankit Agrawal and
+                  SylvainLan and
+                  Michel Pelletier and
+                  Matthew Brett and
+                  Frank Yu and
+                  Daniel M. Pelt and
+                  Saket Choudhary and
+                  Daniele Tricoli and
+                  Check your git settings! and
+                  asnt and
```

Comment:
some weirdness in these two lines, and also `0-tree` as the last author. I guess that's how Zenodo does it, but I think we should remove entries that aren't full names. or at least ping them and add them to the mailmap

Reply:
Hmm. That was exported from Zenodo. I'm not sure what is causing the "Check your git settings!" text to appear there, but will remove it.

Our utils/authors.py maps `asnt` to Alexandre Saint, but does not have an entry for `0-tree` or `jakirkham`. I am pretty sure it is John Kirkham, but don't know for 0-tree. I can ping them and ask. I assume if the user's public GitHub profile doesn't list their name then Zenodo doesn't know how to determine it.

Reply:
Do you happen to know if JOSS typically uses these full contributor lists in the references?  In digital format, it doesn't really cost anything to list everyone, so I had just left everyone in. Alternatively, we could truncate after the first six or so (There are six with >= 10 commits, and 17 with > 1 commit) and list a generic "PyWavelets contributors" for the rest if that is preferable. 

The same issue with GitHub usernames shows up in the ODL reference too

Reply:
> Do you happen to know if JOSS typically uses these full contributor lists in the references? 

I didn't know, but a quick browse suggests that only the first three authors are listed and then it say something like "n.d."